### PR TITLE
GET /users/{term}/{id} endpoint test

### PR DESCRIPTION
**Pull request by chloealee**
#### What does this PR do?
Writes a test to ensure that `GET /users/{term}/{id}` endpoint works correctly with the following terms: `_id`, `mobile`, `email`, and `drupal_id`

#### Where should the reviewer start?
tests/AuthTest.php lines 195-236

#### How should this be manually tested?
`vagrant ssh` into your northstar environment and run `vendor/bin/phpunit` to run all phpunit tests

### CODE REVIEW

File: tests/AuthTest.php [hunk 1]
```diff
@@ -192,4 +192,46 @@ public function testDrupalPasswordChecker()
         $tokenCount = Token::where('key', '=', $data['data']['session_token'])->count();
         $this->assertEquals($tokenCount, 1);
     }
+
+    /**
+     * Tests GET /users/{term}/{id} endpoint works correctly
+     */
+    public function testRetrieveUser() {
+        // User info
+        $user = User::find('5430e850dt8hbc541c37tt3d');
```
Comment (angaither) [inline tests/AuthTest.php]: I don't think this `_id` will work across multiple environments

Comment (chloealee) [inline tests/AuthTest.php]: @angaither testing it based on the first user in the UserTableSeeder.php file! Does that make sense to do/will work for the testing purposes?

**Comment by chloealee** [comment]
@DFurnes mind taking a look when you get the chance?

**Comment by DFurnes** [comment]
Looks good to me! :boat:

**Pull request merged**